packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -268,6 +274,18 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
 
       _.invoke(predefinedProps, 'onBlur', e, predefinedProps);
     },
+    onKeyPress: e => {
+      if (!allowManualInput) {
+        const keyCode = getCode(e);
+
+        if (keyCode === keyboardKey.Enter) {
+          setPreventClosing(true);
+          setOpenState(true);
+        }
+
+        _.invoke(predefinedProps, 'onKeyPress', e, predefinedProps);
+      }
+    },
```

Comment:
Let's move all keyPress handlers to appropriate behaviors, e.g. in https://github.com/microsoft/fluentui/blob/a4898636dfef66f59f63740ab2646c7497410137/packages/fluentui/accessibility/src/behaviors/Menu/menuItemBehavior.ts#L46